lib/view_model/wyre_view_model.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:cake_wallet/entities/wyre_service.dart';
+import 'package:flutter/foundation.dart';
+import 'package:hive/hive.dart';
+import 'package:cake_wallet/entities/order.dart';
+import 'package:cake_wallet/entities/wallet_type.dart';
+import 'package:cake_wallet/store/dashboard/orders_store.dart';
+import 'package:mobx/mobx.dart';
+
+part 'wyre_view_model.g.dart';
+
+class WyreViewModel = WyreViewModelBase with _$WyreViewModel;
+
+abstract class WyreViewModelBase with Store {
+  WyreViewModelBase(this.ordersSource, this.ordersStore,
+      {@required this.walletId, @required this.address, @required this.type})
+      : wyreService = WyreService(walletType: type, walletAddress: address);
```

Comment:
Don't create dependency here, import it by dependency injection container (di.dart).

Reply:
inject dependency through constructor